src/bosh_azure_cpi/lib/cloud/azure/models/config.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+module Bosh::AzureCloud
+  class AzureStackConfig
```

Comment:
Please add related unit test cases.

Reply:
as I said in another PR.
all code path covered in our unit test. so no need to add it.